cmd/create/cluster/cmd.go

```diff
@@ -252,6 +252,25 @@ func run(cmd *cobra.Command, _ []string) {
 		reporter.Errorf("Error getting region: %v", err)
 		os.Exit(1)
 	}
+
+	// Create the AWS client:
+	client, err := aws.NewClient().
+		Logger(logger).
+		Region(region).
```

Comment:
I think this region should be removed from this instantiation. The cloudformation template is applied to a specific region (most likely the default one in the `~/.aws/config` or `$AWS_REGION`) and using the one set for the cluster will just cause it to not be found. See https://github.com/openshift/moactl/pull/92 for more details.

Reply:
hi, @vkareh thanks for the review! appreciated.
Now the region section has been removed.